frontend/src/static/js/main.js

```diff
@@ -160,21 +160,33 @@ window.app = new window.Vue({
       return full;
     },
 
-    deactivateTab() {
+    // handle opening of sidebar //
+    activateTab(tabName) {
+      // changing isTabActive to trigger redrawing of component
       this.isTabActive = false;
       if (document.getElementById('tabs-wrapper')) {
         document.getElementById('tabs-wrapper').scrollTop = 0;
       }
+
+      this.isTabActive = true;
+      this.isCollapsed = false;
+      this.tabType = tabName;
     },
 
     updateTabAuthorship(obj) {
-      this.deactivateTab();
       this.tabInfo.tabAuthorship = Object.assign({}, obj);
+      this.activateTab('authorship');
+    },
+    updateTabZoomin(obj) {
```

Comment:
zoom in? zooming?

Reply:
haha this is right, tab "zoom in"

Reply:
If it is suppose to be "zoom in" then capitalize the `I`.